worker.go

```diff
@@ -92,7 +92,7 @@ func (l *loadBalancer) AddIngress(certificateARNs []string, ingress *kubernetes.
 		l.securityGroup != ingress.SecurityGroup ||
 		l.sslPolicy != ingress.SSLPolicy ||
 		l.loadBalancerType != ingress.LoadBalancerType ||
-		l.http2 != ingress.HTTP2 || l.wafWebACLId != ingress.WafWebACLId {
+		l.http2 != ingress.HTTP2 {
```

Comment:
This `if` has the function of grouping ingresses on load balancers based on all the parameters. If you don't remove the `wafWebACLId` check, then there should not be a need for the  `groupLBsByCurrentWAF()` function below as it's effectively the same thing from what I understand, just implemented in a different way.

Reply:
there was a reason during an interim state for making this change. Basically changes were not properly handled, but I need to check in detail. If possible, then I'll revert this change to make it analogous to the rest of these properites.

Reply:
done